WcaOnRails/db/migrate/20161117085757_cleanup_events_issue96.rb

```diff
@@ -0,0 +1,12 @@
+class CleanupEventsIssue96 < ActiveRecord::Migration
+  def up
+    remove_foreign_key :preferred_formats, column: :event_id
+    remove_foreign_key :competition_events, column: :event_id
```

Comment:
Correct me if I'm wrong, but I believe the reason why we have to set foreign key checks to 0 is because we have data in other tables that reference data in this table? If that is the case, how about doing a cascade on delete operation, then we can get rid of these foreign keys?